core/background_workers.py

```
@@ -0,0 +1,437 @@
+    
+    logger.info("All background workers started successfully")
+
+# Entrypoint for workers (run in separate process if needed)
+if __name__ == "__main__":
+    logging.basicConfig(
+        level=logging.INFO,
+        format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    )
+    
+    loop = asyncio.get_event_loop()
+    loop.create_task(start_background_workers())
```

Ruff (0.14.5)

Store a reference to the return value of `loop.create_task`

(RUF006)